ramls/circulation.raml

```diff
@@ -159,6 +159,26 @@ resourceTypes:
           body:
             text/plain:
               example: "Internal server error"
+  /batch/change-due-date:
```

Comment:
This should be a separate RAML file, since you've declared it in descriptor as a separate API.

Reply:
Done.